src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -610,6 +611,11 @@
             "PeriodicEnergyImported",
             "PeriodicEnergyExported"
         ],
+        "Power Topology": [
+            "FeatureMap",
+            "AvailableEndpoints",
+            "ActiveEndpoints"
```

Comment:
For what it's worth, you don't have to add list-typed attributes here; we know those need to be external at codegen time.  This file is here to communicate that information for non-list-typed attributes, for which there's no way to tell from the XML (kinda; for structs we could, in theory, but there are practical issues with that).

Reply:
OK, removed.